### ISSUE TITLE
Add incremental font-sizing instead of Large, Default, Small

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/cozy-sun-bear#99]
During the accessibility audit, we discovered that epub.js disables zooming and text-scaling in the `<iframe>` that contains the EPUB content, causing a WCAG 2.0AA critical violation.

```
<meta name="viewport" content="width=1369, height=703, initial-scale=1, minimum-scale=1, maximum-scale=1, user-scalable=no">
```

The recommended solution is to remove the `user-scalable` attribute and set a `maximum-scale` higher than 2. However, doing so would cause severe display issues for the page-by-page view of the EPUB.  In addition, while text size does increase when a user zooms the entire browser, it does not increase when zooming only text on the page.

One way to provide low-vision or partial sight readers with larger text would be to give them additional options for increasing/decreasing font-size in the Preferences widget. For example the ability to + or - font size instead, which would increase the font-size by some increment that we determine to be sufficient and user-friendly.

Before proceeding, we should determine if this is actually solving the issue that is raised when zooming is disabled.

**Comment by respinos** [comment]
The modifications to the text size preferences is addressed in `keyboard-and-preferences`

**Comment by jmcglone** [comment]
Resolved by PR #150
